plugins/services/src/js/structs/Framework.js

```diff
@@ -20,6 +20,32 @@ module.exports = class Framework extends Application {
     return this.get("slave_ids");
   }
 
+  getFullId() {
```

Comment:
Very related to the create [createService()](https://github.com/dcos/dcos-ui/pull/2339/files#diff-361072f18fbf1e35155c80dd51343244R230) helper in my system test PR. In `createService()` I made it enforce the leading slash (even though the backend would work without slash also, I think).

**SO**, I agree with Mat, would be nice to be consistent with enforcing that slash.